src/components/NewChip/NewChip.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { css, cx } from '@emotion/css';
+import { tokens } from '../../theme/tokens';
+
+export const chipColors = ['primary', 'grey', 'error', 'warning', 'info'] as const;
```

Comment:
Can you check if there's an enum or a constant for colors

Reply:
Missed that 👍